### PR TITLE
docs: Updates to the API keys section

### PR DESCRIPTION
**Pull request by WeaponXII**
This updates the API keys section to explain how to issue and revoke API keys, and also shows an example of using a curl script to add to the API keys

### CODE REVIEW

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (pkafei) [inline content/docs/get-started-with-neon/using-api-keys.md:16]: I know you didn't add this @WeaponXII, but I find the whole section on the `An API key provides access...` confusing. The information is very contextual and may not be the first thing a user needs to know in order to get started. I do have some questions:

1. Is it mission critical for users to know that API keys cannot be scoped to specific Projects? If so this statement needs more details.

2. We should consider rewriting this section to improve readability. `Provision multiple API Keys` is an accurate statement, but it might be too jargony for a "Getting Started Section".

**Review by pkafei — DISMISSED** [review]

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:17]:
```suggestion
An API key provides access to any action available to the user. An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked. Neon users can generate multiple API keys.
```

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:24]: It's not the Settings, but Account.
Additionally, not sure if we should call this a 'button'.
Screenshot 2022-08-02 at 18 53 44

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:26]: I think this is actually a bad example of naming things..
You can never guess that Test Key is for test environment, WFIW it could be something for trying out the API keys.

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:29]: > Neon does not store this key for you

this is ambiguous, shall we reword?

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:31]: Curious to hear your thinking, why this needs to be stressed and emphasized with 'Remember, '.

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:35]: same as in https://github.com/neondatabase/website/pull/102/files#r935822466

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:37]: should we elaborate on what would be the response for those?
@antonyc , @ololobus , how informative is our error?

Comment (stepashka) [reply content/docs/get-started-with-neon/using-api-keys.md:37]: ah, this is actially outilned below.

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:59]: should we outline that this tool is installed separately and is not part of Neon?

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:43]: Actually, we're getting more info, not just the IDs. 
```suggestion
Let’s look at how to make a `curl` request using your Neon API key. We're going to use the `projects` endpoint to get a list of the projects in an account.
```
WDYT about this suggestion?

Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md:43]: This makes the sentence a lot clearer

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md:59]: Yes, it is a little ambiguous. I'll add a link to it for clarity

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md]: What do you think of "Testing Environment" as the name for this example key?

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md]: > You need to store your key in a safe location after generating it, you will not be able to access this value again after leaving your `Developer Settings`.
Is this phrasing clearer?

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md:31]: To tie this section to the earlier statement about API keys remaining valid forever, highlight the statement for readers who might have skimmed the earlier section because they wanted to rush to actionable steps, and also segue into the next section by reminding the user that revoking API keys is important.

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md]: This has now been updated

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (pkafei) [reply content/docs/get-started-with-neon/using-api-keys.md]: @WeaponXII Did you address this comment and delete the word `button`?

Comment (WeaponXII) [reply content/docs/get-started-with-neon/using-api-keys.md]: @pkafei yes, my last commit resolved this

Comment (pkafei) [reply content/docs/get-started-with-neon/using-api-keys.md]: Thanks!

File: content/docs/get-started-with-neon/using-api-keys.md
Comment (stepashka) [inline content/docs/get-started-with-neon/using-api-keys.md:26]:
```suggestion
4. Choose a unique name that will help you remember what this key is for.
```

**Review by stepashka — DISMISSED** [review]
Suggestion + Approve in this review

**Comment by WeaponXII** [comment]
@stepashka I need another approval after accepting the suggestion before I can merge

**Review by stepashka — APPROVED** [review]

**Pull request merged**
